gmso/external/convert_hoomd.py

```diff
@@ -292,17 +361,93 @@ def _parse_particle_information(
         site.name if site.atom_type is None else site.atom_type.name
         for site in top.sites
     ]
-    unique_types = sorted(list(set(types)))
-    typeids = np.array([unique_types.index(t) for t in types])
-    masses = list()
-    charges = list()
-    for site in top.sites:
-        masses.append(
+    masses = u.unyt_array(
+        [
             site.mass.to_value(base_units["mass"])
             if site.mass
             else 1 * base_units["mass"]
-        )
-        charges.append(site.charge if site.charge else 0 * u.elementary_charge)
+            for site in top.sites
+        ]
+    )
+    charges = u.unyt_array(
+        [site.charge if site.charge else 0 * u.elementary_charge for site in top.sites]
+    )
+    moits = [[1, 0, 0] for i in xyz]
```

Comment:
```suggestion
    moment_of_inertias = [[1, 0, 0] for i in xyz]
``` 
Just for clarify on what this variable is,